curvefs/src/mds/topology/topology.cpp

```diff
@@ -351,18 +351,32 @@ TopoStatusCode TopologyImpl::UpdateMetaServerSpace(const MetaServerSpace &space,
         auto it = metaServerMap_.find(id);
         if (it != metaServerMap_.end()) {
             WriteLockGuard wlockMetaServer(it->second.GetRWLockRef());
-            diffCapacity = space.GetDiskCapacity() -
-                           it->second.GetMetaServerSpace().GetDiskCapacity();
+            diffCapacity = space.GetDiskThreshold() -
+                           it->second.GetMetaServerSpace().GetDiskThreshold();
             int64_t diffUsed = space.GetDiskUsed() -
                                it->second.GetMetaServerSpace().GetDiskUsed();
-            int64_t diffMemory =
+            int64_t diffDiskMinRequire =
+                space.GetDiskMinRequire() -
+                it->second.GetMetaServerSpace().GetDiskMinRequire();
+            int64_t diffMemoryThreshold =
+                space.GetMemoryThreshold() -
+                it->second.GetMetaServerSpace().GetMemoryThreshold();
+            int64_t diffMemoryUsed =
                 space.GetMemoryUsed() -
                 it->second.GetMetaServerSpace().GetMemoryUsed();
+            int64_t diffMemoryMinRequire =
+                space.GetMemoryMinRequire() -
+                it->second.GetMetaServerSpace().GetMemoryMinRequire();
             it->second.SetMetaServerSpace(space);
-            if (diffCapacity != 0 || diffUsed != 0 || diffMemory != 0) {
+            if (diffCapacity != 0 || diffUsed != 0 || diffDiskMinRequire != 0 ||
+                diffMemoryThreshold != 0 || diffMemoryUsed != 0 ||
+                diffMemoryMinRequire != 0) {
                 DVLOG(6) << "update metaserver, diffCapacity = " << diffCapacity
                          << ", diffUsed = " << diffUsed
-                         << ", diffMemory = " << diffMemory;
+                         << ", diffDiskMinRequire = " << diffDiskMinRequire
+                         << ", diffMemoryThreshold = " << diffMemoryThreshold
+                         << ", diffMemoryUsed = " << diffMemoryUsed
+                         << ", diffMemoryMinRequire = " << diffMemoryMinRequire;
```

Comment:
The line370 'it->second.SetMetaServerSpace(space);' should put after line379 ?

Reply:
done